tools/build_cdn.js

```diff
@@ -141,7 +141,8 @@ function installStyles() {
     if (stat.isDirectory()) return;
 
     if (file.endsWith(".css")) {
-      installCleanCSS(`./src/styles/${file}`, `styles/${file.replace(".css", ".min.css")}`);
+      installCleanCSS(`./src/styles/${file}`, `styles/${file}`, { minify: false });
+      installCleanCSS(`./src/styles/${file}`, `styles/${file.replace(".css", ".min.css")}`, { minify: true });
```

Comment:
Ok, this is non-breaking because we're adding a NEW file, not changing behavior.